pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/PropertiesToMapTranslator.groovy

```diff
@@ -52,16 +57,18 @@ public class PropertiesToMapTranslator implements MethodMissingWrapper, Serializ
             argValue = args[0]
         }
 
-        if (resolveCredentials && s == "credentials") {
-            if (args.length == 1) {
-                String id = "${args[0]}"
-
-                RunWrapper currentBuild = script.getProperty("currentBuild")
+        if (resolveEnvironmentContributors) {
+            def retVal
+            if (argValue != null) {
```

Comment:
This is actually an interesting question in the context of #110, where we stop actually evaluating the `PropertiesToMapTranslator` entirely - with that, we may be able to switch to `@Symbol("properties")` after all.